mmv1/products/compute/api.yaml

```diff
@@ -7357,13 +7356,34 @@ objects:
           selected availability domain will be provided to the Partner via the
           pairing key so that the provisioned circuit will lie in the specified
           domain. If not specified, the value will default to AVAILABILITY_DOMAIN_ANY.
+        values:
+          - :AVAILABILITY_DOMAIN_1
+          - :AVAILABILITY_DOMAIN_2
+          - :AVAILABILITY_DOMAIN_ANY
+      - !ruby/object:Api::Type::NestedObject
+        name: 'partnerMetadata'
+        description: |
+          Informational metadata about Partner attachments from Partners to display
+          to customers. Output only for for PARTNER type, mutable for PARTNER_PROVIDER,
+          not available for DEDICATED.
+        properties:
```

Comment:
Can all of these subfields be made required? Is it valid to specify only some of them for a PARTNER_PROVIDER?

Reply:
I believe they're all required sub-fields. I will update that